tests/drivers/pwm/pwm_loopback/boards/disco_l475_iot1.overlay

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 Tilmann Unte
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pwm/pwm.h>
+
+/ {
+	pwm_loopback_0 {
+		compatible = "test-pwm-loopback";
+		/* first index must be a 32-Bit timer */
+		pwms = <&pwm2 1 0 PWM_POLARITY_NORMAL>,
+			<&pwm15 1 0 PWM_POLARITY_NORMAL>;
+	};
+};
+
+/* 32-Bit timers */
+&timers2 {
+	status = "okay";
+	pwm2: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim2_ch1_pa5>; /* CN1 D13 */
+	};
+};
+
+&timers5 {
+	status = "okay";
+	pwm5: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim5_ch1_pa0>; /* CN3 D1 */
+	};
+};
+
+/* 16-Bit timers */
+&timers3 {
+	status = "okay";
+	pwm3: pwm {
+		status = "okay";
+		st,prescaler = <255>;
```

Comment:
Should be sync'ed with https://github.com/zephyrproject-rtos/zephyr/pull/39628.
This won't trigger any failure, but we should avoid having both implementations in tree once #39628 is merged

Reply:
I worked out the minor nitpicks. Awaiting #39628 now, thank you for pointing it out to me.

Reply:
@TilmannUnte https://github.com/zephyrproject-rtos/zephyr/pull/39628 has just been merged.

Reply:
All done!